updated to match the new ui (#1528)
keycloak/keycloak-documentation: authorization_services/topics/hello-world-create-realm.adoc

+
The user list page displays where you can create a user.

ifeval::[{project_community}==true]
. Click *Create user*.
endif::[]
ifeval::[{project_product}==true]
. Click *Add User*.
endif::[]

. Complete the *Username*, *Email*, *First Name*, and *Last Name* fields.

> . Toggle *User Enabled* to *ON*

Reviewer: ```suggestion
. Toggle *User Enabled* to *ON*.
```

ifeval::[{project_community}==true]
. Click *Create*.
endif::[]
ifeval::[{project_product}==true]
. Click *Save*.
endif::[]

+
.Add User

> image:{project_images}/getting-started/hello-world/create-user.png[alt="Add User"]

Reviewer: Can you slightly increase this screen shot so that "identity providers" in the menu is not cut off?

> . Complete the *New Password* and *Password Confirmation* fields with a password and click the *Temporary* switch to *OFF*.

Reviewer: ```suggestion
. Complete the *New Password* and *Password Confirmation* fields and toggle *Temporary* to *OFF*.
```
